Dialog: examples and text improvements (#22671)
mdn/content: files/en-us/web/api/htmldialogelement/returnvalue/index.md

## Examples

> The following example displays a button to open a {{htmlelement("dialog")}} containing a form via the `showModal()` method. The script assigns the `returnvalue` an initial value. From there, the confirm button submits the form with validation and the _X__ button submits the form without validation. Submitting a form with a `method="dialog"` closes the the dialog and sets the returnValue to the name, if any, of the submit button. The reset button has an even handler that closes the dialog. It has no impact on the returnValue. Neither does closing the dialog with the <kbd>Esc</kbd> key.

Reviewer: If referring to `returnValue` then markup as code. If you mean "return value" then that should be split.

```suggestion
The following example displays a button to open a {{htmlelement("dialog")}} containing a form via the `showModal()` method. The script assigns the `returnvalue` an initial value. From there, the confirm button submits the form with validation and the _X__ button submits the form without validation. Submitting a form with a `method="dialog"` closes the the dialog and sets the `returnValue` to the name, if any, of the submit button. The reset button has an event handler that closes the dialog. It has no impact on the `returnValue`. Neither does closing the dialog with the <kbd>Esc</kbd> key.
```

> The following example displays a button to open a {{htmlelement("dialog")}} containing a form via the `showModal()` method. The script assigns the `returnvalue` an initial value. From there, the confirm button submits the form with validation and the _X__ button submits the form without validation. Submitting a form with a `method="dialog"` closes the the dialog and sets the returnValue to the name, if any, of the submit button. The reset button has an even handler that closes the dialog. It has no impact on the returnValue. Neither does closing the dialog with the <kbd>Esc</kbd> key.

Author: ```suggestion
The following example displays a button to open a {{htmlelement("dialog")}} containing a form via the `showModal()` method. The script assigns the `returnvalue` an initial value. From there, the confirm button submits the form with validation and the _X__ button submits the form without validation. Submitting a form with a `method="dialog"` closes the the dialog and sets the return value to the `name`, if any, of the submit button. The reset button has an event handler that closes the dialog. It has no impact on the returnValue. Neither does closing the dialog with the <kbd>Esc</kbd> key.
```

> The following example displays a button to open a {{htmlelement("dialog")}} containing a form via the `showModal()` method. The script assigns the `returnvalue` an initial value. From there, the confirm button submits the form with validation and the _X__ button submits the form without validation. Submitting a form with a `method="dialog"` closes the the dialog and sets the return value to the `name`, if any, of the submit button. The reset button has an event handler that closes the dialog. It has no impact on the returnValue. Neither does closing the dialog with the <kbd>Esc</kbd> key.

Author: ```suggestion
The following example displays a button to open a {{htmlelement("dialog")}} containing a form via the `showModal()` method. The script assigns the `returnvalue` an initial value. From there, the confirm button submits the form with validation and the _X__ button submits the form without validation. Submitting a form with a `method="dialog"` closes the the dialog and sets the return value to the `name`, if any, of the submit button. The reset button has an event handler that closes the dialog; it has no impact on the `returnValue`. Neither does closing the dialog with the <kbd>Esc</kbd> key.
```
